README.md

```diff
@@ -346,8 +346,36 @@ lookup (for physical measurements and biospecimen donation) as well as a
 Participant Work Queue. Any of the above parameters can be provided as a query parameter to do
 an exact match. Examples:
 
-    GET /ParticipantSummary?hpoId=PITT
-    GET /ParticipantSummary?hpoId=PITT&state=PIIState_MA
+    GET /ParticipantSummary?awardee=PITT
+    GET /ParticipantSummary?awardee=PITT&state=PIIState_MA
+
+    GET /ParticipantSummary?organization=PITT_UPMC 
+    GET /ParticipantSummary?site=hpo-site-UPMC
+
+The participant summary API supports filtering and sorting on last modified time.
+The order results are returned is...
```

Comment:
Two things: our parameter / field names are normally camel case... so it's lastModified not last_modified.

Secondly, the way to request sorting in this API is passing a _sort parameter.

Thirdly, we may prefer to have the pagination for this API provided with a _token parameter as is currently supported, which will encode the timestamp as an opaque string... 

If that's the case, then the API would look like:

GET /ParticipantSummary?awardee=PITT&_sort=lastModified
GET /ParticipantSummary?awardee=PITT&_sort=lastModified&_token=abciufeshwg

Reply:
Note that the normal way of making pagination tokens in this API is here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/base_dao.py#L164

and the way of parsing them is here:

https://github.com/all-of-us/raw-data-repository/blob/master/rest-api/dao/base_dao.py#L259

However you'll need to handle things a bit differently in this API since the timestamp needs to be adjusted backwards to account for clock skew, etc., and you probably only want the adjusted timestamp in the token (and not also a participant ID.)

Reply:
We will also need to indicate that the same participant summary (with the same data) can come back multiple times when syncing; the client should handle that gracefully.